modules/ROOT/pages/apis/tinymce.addonmanager.adoc

```diff
@@ -0,0 +1,130 @@
+= tinymce.AddOnManager
+:navtitle: tinymce.AddOnManager
+:description: This class handles the loading of add-ons and their language packs. ThemeManager and PluginManager are instances of AddOnManager, and manage themes and plugins.
+:keywords: add, get, load, requireLangPack
+:moxie-type: api
+
+This class handles the loading of add-ons and their language packs. ThemeManager and PluginManager are instances of AddOnManager, and manage themes and plugins.
+
+[[summary]]
+== Summary
+
+[[methods-summary]]
+=== Methods
+[cols="2,5,1",options="header"]
+|===
+|Name|Summary|Defined by
+|xref:#add[add()]|Adds a instance of the add-on by it's short name.|`xref:apis/tinymce.addonmanager.adoc[AddOnManager]`
+|xref:#get[get()]|Returns the specified add on by the short name.|`xref:apis/tinymce.addonmanager.adoc[AddOnManager]`
+|xref:#load[load()]|Loads an add-on from a specific url.|`xref:apis/tinymce.addonmanager.adoc[AddOnManager]`
+|xref:#requireLangPack[requireLangPack()]|Loads a language pack for the specified add-on.|`xref:apis/tinymce.addonmanager.adoc[AddOnManager]`
+|===
+
+[[methods]]
+== Methods
+
+[[add]]
+=== add()
+[source, javascript]
+----
+add(id: String, addOn: tinymce.Theme | tinymce.Plugin): tinymce.Theme | tinymce.Plugin
+----
+Adds a instance of the add-on by it's short name.
```

Comment:
```suggestion
Adds an instance of the add-on by its short name.
```
Typo corrections.

1. The indefinite article is *an* when the following sound is a vowel.
2. *Its* is the possessive pronoun. *It’s* is the abbreviation. FWIW, I remember this ‘exception’ by remembering that no possessive pronouns take an apostrophe (so: his, hers, theirs, its).

Reply:
@brian-forte I think you missed an important part of the README updates here:
> **Note:** The API documentation should never be edited manually.

So as noted, this cannot be changed like this and any of these changes would need to be done on the TinyMCE source.

Reply:
I'd suggest logging these changes in a Jira and getting them fixed that way instead.